Makefile

```
@@ -0,0 +1,30 @@
+install-user-vault: build # install plugin to user vault
+	-mkdir -p $(VAULT)/.obsidian/plugins/neighbouring-files/
+	-cp -rf $(FILES) $(VAULT)/.obsidian/plugins/neighbouring-files/
+
+install-test-vault: build # install plugin to test vault
+	-mkdir -p ./vault/.obsidian/plugins/neighbouring-files/
+	-cp -rf $(FILES) ./vault/.obsidian/plugins/neighbouring-files/
+
+changeset:
+	npx changeset
+
+release:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)